apps/console/src/app.tsx

```diff
@@ -34,6 +34,7 @@ import { I18nextProvider } from "react-i18next";
 import { useDispatch, useSelector } from "react-redux";
 import { Redirect, Route, Router, Switch } from "react-router-dom";
 import { initializeAuthentication } from "./features/authentication";
+import { AuthenticateUtils } from "./features/authentication/utils/authenticate-utils";
```

Comment:
```suggestion
import { AuthenticateUtils } from "./features/authentication/utils";
```

Reply:
Addressed in https://github.com/wso2/identity-apps/pull/1732/commits/6ce4484d6f7bfe758f7f22f590a45a499e21d508